package.json

```diff
@@ -47,15 +48,15 @@
     "eslint-plugin-jsx-a11y": "^0.6.2",
     "eslint-plugin-react": "^4.3.0",
     "mocha": "^2.4.5",
-    "react": "^15.0.1",
-    "react-addons-test-utils": "^15.0.1",
-    "react-dom": "^15.0.1",
+    "react": "^16.5.2",
+    "react-addons-test-utils": "^15.6.2",
+    "react-dom": "^16.5.2",
     "rimraf": "^2.5.2",
     "sinon": "^1.17.3",
     "webpack": "^1.13.0"
   },
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "react": "^16.2.0",
```

Comment:
Would it be possible to make this 15 OR 16 so that this wouldn't require a major version bump for the package itself?